bundles/org.eclipse.vorto.editor.infomodel.web/src/org/eclipse/vorto/editor/infomodel/web/ServerLauncher.xtend

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ * Copyright (c) 2014 Bosch Software Innovations GmbH and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
+ * The Eclipse Public License is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * The Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * Contributors:
+ * Bosch Software Innovations GmbH - Please refer to git log
+ *
+ *******************************************************************************/
+/*
+ * generated by Xtext
+ */
+package org.eclipse.vorto.editor.infomodel.web
+
+import java.net.InetSocketAddress
+import org.eclipse.jetty.annotations.AnnotationConfiguration
+import org.eclipse.jetty.server.Server
+import org.eclipse.jetty.util.log.Slf4jLog
+import org.eclipse.jetty.webapp.MetaInfConfiguration
+import org.eclipse.jetty.webapp.WebAppContext
+import org.eclipse.jetty.webapp.WebInfConfiguration
+import org.eclipse.jetty.webapp.WebXmlConfiguration
+
+/**
+ * This program starts an HTTP server for testing the web integration of your DSL.
+ * Just execute it and point a web browser to http://localhost:8080/
+ */
+class ServerLauncher {
```

Comment:
Can you remove this class as it is not needed in the future due to usage of springboot. Can you also verify and clean unneeded dependencies in the pom.xml ?
Thanks!